src/HttpCache/VarnishPurger.php

```diff
@@ -24,14 +24,43 @@
  */
 final class VarnishPurger implements PurgerInterface
 {
+    private $maxHeaderLength;
     private $clients;
 
     /**
      * @param ClientInterface[] $clients
+     * @param int               $maxHeaderLength
```

Comment:
Remove useless phpdoc (use typehint instead).

Reply:
ok for the typehint, but keeping the phpdoc avoid warning from IDE like phpstorm

Reply:
phpdoc removed anyway :)